### PR TITLE
Support master key in the commands

### PR DESCRIPTION
**Pull request by Toreno96**
- `init` command
- `key` command
- support keyring

### CODE REVIEW

File: onetimepass/otp.py [hunk 1]
```diff
@@ -1,22 +1,27 @@
...
-    else:
-        return True
+try:
+    import keyring  # noqa 'keyring' in the try block with 'except ImportError' should also be defined in the except block
+except ImportError:
+    KEYRING_INSTALLED = False
+else:
+    KEYRING_INSTALLED = True
+    KEYRING_PARAMS = {
+        "service_name": "onetimepass",
```
Comment (khanek) [inline onetimepass/otp.py]: What do you think to move these params to settings.py file?

File: onetimepass/otp.py [hunk 2]
```diff
@@ -64,26 +69,44 @@ def show(ctx: click.Context, alias: str):
...
-        click.echo(key_)
+        if keyring_ and KEYRING_INSTALLED:
+            keyring_set(password=key_)
+
+        if not quiet:
+            click.echo(key_)
+    except click.ClickException as e:
+        raise e
+    except Exception as e:
+        breakpoint()
```
Comment (khanek) [inline onetimepass/otp.py]: breakpoint :)

**Review by khanek — CHANGES_REQUESTED** [review]

File: onetimepass/otp.py (hunk 1 above)
Comment (Toreno96) [reply onetimepass/otp.py]: 👍

File: onetimepass/otp.py (hunk 2 above)
Comment (Toreno96) [reply onetimepass/otp.py]: thanks 🙈

**Review by khanek — APPROVED** [review]

**Pull request merged**